### ISSUE TITLE
Load wasm into the netjs

### ISSUE THREAD

**Issue by dlpigpen** [nestjs/nest#9276]
### Is there an existing issue that is already proposing this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe it

I am wondering is there any way to support using a wasm file directly instead of using webpack loader. I am writing some API methods that are going to wasm but it is too difficult to load a file and call as a module. Especially interacting with typescript.

### Describe the solution you'd like

we can integrate the assembly script loader plugin to import wasm file into our framework https://www.npmjs.com/package/@assemblyscript/loader

### Teachability, documentation, adoption, migration strategy

- https://www.assemblyscript.org/
[- ](https://www.assemblyscript.org/loader.html#installation)

### What is the motivation / use case for changing the behavior?

- Support the modern language that going to use too much in coming days

**Comment by jmcdo29** [comment]
Correct me if I'm wrong, but isn't WASM specifically to write JS that will be ran in the browser but from a different language? What are you doing with WASM in NestJS?

**Comment by dlpigpen** [comment]
> Correct me if I'm wrong, but isn't WASM specifically to write JS that will be ran in the browser but from a different language? What are you doing with WASM in NestJS?
Yeah. I am a blockchain developer, I am using netjs a backend service. There are some ecosystems using the wasm files as a contract exported. Especially as the trending now we are going to use the wasm files to speed up the loading websites and security network...

if Netjs can support and load them easily. It will be a killer feature.

**Comment by kamilmysliwiec** [comment]
There should be no difference in using WASM in NestJS vs using WASM in pure Node.js (without NestJS).

Please, use our [Discord](https://discord.gg/G7Qnnhy) channel (support) for such questions. We are using GitHub to track bugs, feature requests, and potential improvements.
